pangeo-notebook/packages.txt

```diff
@@ -411,10 +419,10 @@ notebook==7.0.6
 notebook-shim==0.2.3
 nspr==4.35
 nss==3.94
-numba==0.57.1
+numba==0.58.1
 numbagg==0.6.0
 numcodecs==0.11.0
-numpy==1.24.4
+numpy==1.26.0
```

Comment:
Somewhat large jump in `numpy` version from 1.24 to 1.26. Should be ok though hopefully :crossed_fingers: